news/2025/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer.md

```diff
@@ -0,0 +1,289 @@
+---
+layout: post
+title: "The Followpoint: Spectator, the osu!catch Mapping Pioneer"
+date: 2025-04-30 09:00:00 +0000
+series: The Followpoint
+---
+
+Have you ever looked at osu! interviews and thought to yourself: "Hey, aren't there too little mapper interviews"? Well, today is your lucky day, as we are interviewing Spectator, one of the oldest osu!catch mappers out there!
+
+![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/banner.jpg)
+
+You know how there are players, mappers and [Beatmap Nominators](/wiki/People/Beatmap_Nominators), right? And it's not exactly common for one to do two, or even all three of these activities, but [Spectator](https://osu.ppy.sh/users/702598) is exactly this kind of person! While being a top player [peaking at rank #2](https://web.archive.org/web/20120701011805/http://osu.ppy.sh:80/p/pp/?m=2) and winning multiple [osu!catch World Cups](/wiki/Tournaments/CWC), he's also a mapper with many maps and difficulties of his in both [Ranked](/wiki/Beatmap/Category#ranked) and [Loved](/wiki/Beatmap/Category#loved) categories, and of course a Beatmap Nominator, and not a regular one, he's an [Elite Nominator](/wiki/People/Elite_Nominators), personally nominating 145 beatmaps in 2024!
+
+So now that you have an idea how big of community figure Spectator is for osu!catch — let's get to the interview [me](https://osu.ppy.sh/users/18152711) and [-Isla-](https://osu.ppy.sh/users/17745759) conducted with him!
```

Comment:
credit is both at the top and bottom of the post, and we very rarely refer to the authors in first person

```suggestion
Now that you have an idea how big of a community figure Spectator is for osu!catch, let's get to hear what insights he has to bring! This time, [-Isla-](https://osu.ppy.sh/users/17745759) is joining in with questions as well.
```

Reply:
Alright, gotcha! I wasn't sure how I should write this, so thank you for clarifying.